README.md

```diff
@@ -12,8 +12,6 @@
   - [Deployment](#deployment)
```

Comment:
- [ ] It would be nice if you fix your Deployment link because it is broken 😨 Kindly fix it or remove it **(if you don't have any live section for this link)** to make your readme descriptive and professional 👍

Reply:
Thanks, I will do that.